### PR TITLE
Feature/my communities filter

### PR DESCRIPTION
**Pull request by Zoeshwa**

### CODE REVIEW

**Comment by Zoeshwa** [comment]
i change the css - tell me what you think :)

File: apps/ui/src/Communities/MyEntitiesFilterButton.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { Box, FormControlLabel, Switch } from '@mui/material';
+
+export interface MyEntitiesFilterButtonProps {
+  lable: string;
+  setShowMyEntities: (isOn: boolean) => void;
+  showMyEntities: boolean;
+  filter: (showMyEntities: boolean) => void;
+}
+
+const MyEntitiesFilterButton: React.FC<MyEntitiesFilterButtonProps> = ({
```
Comment (Kovler10) [inline apps/ui/src/Communities/MyEntitiesFilterButton.tsx:10]: this component is a bit weird. cannot be reused much

File: apps/ui/src/Components/PageHeader/PageHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import { Box, Button, Typography } from '@mui/material';
+import { Menu } from '../Menu/Menu';
+import { Menu as MenuIcon } from '@mui/icons-material';
+
+export interface PageHeaderProps {
+  title: string;
+}
+
+const PageHeader: React.FC<PageHeaderProps> = ({ title }) => {
```
Comment (Kovler10) [inline apps/ui/src/Components/PageHeader/PageHeader.tsx:9]: This is great! can you add this to other pages as well?

**Review by Kovler10 — CHANGES_REQUESTED** [review]
Nice! Make sure this won't conflict with some of your other PR's

**Review by Kovler10 — APPROVED** [review]

**Pull request merged**